src/app/core/rpc/rpc.service.ts

```diff
@@ -337,5 +336,12 @@ export class RPCService {
     // A poll only for address changes, triggered from the GUI!
     this._callOnAddress.forEach(this._pollCall.bind(this));
   }
+
+  /** Send Notification to the backend */
```

Comment:
Not in the right place, notifications in the rpc service?

Reply:
Rpc using rpx where all stuffs written related to communication between frontend and backend(electron), That is the reason I just added another method which called from rpc service and emit the message to main.js